gnomad/utils/vcf.py

```diff
@@ -297,31 +343,56 @@
             " deleterious."
         ),
     },
-    "revel_score": {
+    "revel_max": {
         "Number": "1",
         "Description": (
-            "dbNSFP's Revel score from 0 to 1. Variants with higher scores are"
+            "The maximum dbNSFP's Revel score at a site's MANE Select or canonical"
+            " transcript. Scores ranges from 0 to 1. Variants with higher scores are"
             " predicted to be more likely to be deleterious."
```

Comment:
Based on my understanding, REVEL is independent of dbNSFP. Also I downloaded directly from REVEL's website. 
dbNSFP compiles all kinds of prediction scores. 
```suggestion
            "The maximum REVEL score at a site's MANE Select or canonical transcript."
             "It's an ensemble score for predicting the pathogenicity of missense "
             "variants (based on 13 other variant predictors). Scores ranges from 0 to 1. "
             "Variants with higher scores are predicted to be more likely to be "
             "deleterious."
```